[IMP] mrp: documentation major update (#1460)
odoo/documentation: content/applications/inventory_and_mrp/manufacturing/management/alternative_wc.rst

================================
Alternative Work Centers
================================

For many manufacturing companies, a common issue is to have to
manufacture, at the same time, several products usually produced at the
same work center. If in practice, employees can manufacture the goods at
another work center, it is not that simple to have visibility on which 

> work center is busy and plan ressources accordingly in real-time.  

Reviewer: ok

For that, you need to keep track of the different work orders: which work 
center has been used and when, so you can schedule orders and maintenance

> accordingly. With Odoo, you can configure alternative work centers to solve this issue. 

Reviewer: Modified it a little bit further as it sounded repetitive with the first section.

This way, your process becomes more efficient, and less non-productive

> time is lost.

Reviewer: removed the 'productive' concept, as this is rather related to an operation versus the initial planning concept which we are targeting here.

================================
Alternative Work Centers
================================

Manufacturing several different products simultaneously at the same work center
is a common issue for manufacturing companies. Although in practice, employees can 
manufacture the goods at another work center instead of waiting for one to be available, 
it's not always simple to have visibility on which work center is busy and plan resources 
accordingly in real-time.

With Odoo, you can configure alternative work centers to solve this issue. This automatically checks 

> which work center is busy and which equivalent ones are free, and schedules orders and maintenance accordingly.

Reviewer 2: Take care to [limit the lines' length to 100 characters](https://www.odoo.com/documentation/master/contributing/documentation/rst_guidelines.html#start-a-new-line-before-the-100th-character).
```suggestion
which work center is busy and which equivalent ones are free, and schedules orders and maintenance
accordingly.
```

This way, your process becomes more efficient, as work center utilisation is optimised. 

Configure your Work Centers
===========================

> First, go to the Manufacturing app > Configuration> Settings. Then, enable the Work Orders

Reviewer 2: Use the `menuselection` directive. See H2 of https://www.odoo.com/documentation/master/contributing/documentation/content_guidelines.html#document-structure

feature and hit save.

> .. image:: media/BOM_3.png

Reviewer 2: Make sure that all new (or reworked) images have:
- a consistent file name
- the right location: place the file inside a folder of the same name as the doc file referring to it (pick the folder that makes the most sense if two pages are referring to the same image)
- an ALT tag

See https://www.odoo.com/documentation/master/contributing/documentation/content_guidelines.html#images

Reviewer 2: Also, please run `pngquant` on all these images to reduce their weight by a good 2/3rd.

================================
Alternative Work Centers
================================

Manufacturing several different products simultaneously at the same work center
is a common issue for manufacturing companies. Although in practice, employees can 
manufacture the goods at another work center instead of waiting for one to be available, 
it's not always simple to have visibility on which work center is busy and plan resources 
accordingly in real-time.

With Odoo, you can configure alternative work centers to solve this issue. This automatically
checks which work center is busy and which equivalent ones are free, and schedules orders and
maintenance accordingly.

This way, your process becomes more efficient, as work center utilisation is optimised. 

Configure your Work Centers
===========================

> First, go to the :menuselection:`Manufacturing app > Configuration > Settings`. Then, enable

Reviewer 2: `menuselection` needs selection items to be separated by `-->` and not `>`. Otherwise, you are creating a selection of a single item.

the Work Orders feature and hit save.

.. image:: media/bom_3.png
    :align: center
    :alt: work order settings ticked

> Now, go to the *Work Centers* menu under *Configuration* and create

Reviewer 2: ```suggestion
Now, go to :menuselection:`Configuration --> Work Centers` and create
```

two new work centers. Add the second work center as an alternative to
the first one and vice versa.

.. image:: media/wc_alternate1.png
    :align: center

> Next, go to Products > Bills of Materials, choose your product, 

Reviewer 2: ```suggestion
Next, go to :menuselection:`Products --> Bills of Materials`, choose your product, 
```

and add these work centers to the operations of your BOM.

.. image:: media/wc_alternate2.png
    :align: center

Create your Work Orders
=======================

Now that everything is configured, you can create your *Work Orders*.

> Go to **Operations** > **Manufacturing Orders**, and hit *Create*. There, add

Reviewer 2: ```suggestion
Go to :menuselection:`Operations --> Manufacturing Orders`, and hit **Create**. There, add
```

> Then, enable the *Work Orders* feature and hit save.

Reviewer: ok

.. image:: media/BOM_3.png
    :align: center

Now, go to the *Work Centers* menu under *Configuration* and create
two new work centers. Add the second work center as an alternative to
the first one and vice versa.

.. image:: media/WC_Alternate1.png
    :align: center

> The next step is to add these work centers to the Operations of your BOM,

Reviewer: ok
